cmd/zb/perf.go

```diff
@@ -198,7 +200,17 @@ func updateStats(summary *statsSummary, record statsRecord) {
 	summary.latencies = append(summary.latencies, record.latency)
 }
 
-func printStats(requests int, summary *statsSummary) {
+type cicdTestSummary struct {
+	Name  string      `json:"name"`
+	Unit  string      `json:"unit"`
+	Value interface{} `json:"value"`
+	Range string      `json:"range,omitempty"`
```

Comment:
How is this field used?

Reply:
Specific to the ci-cd workflow we are using. Range says from the historical values +- how much it can deviate.

Reply:
Ok, now I understand